### DISCUSSION TITLE
About the website XFEM case acquisition

### DISCUSSION THREAD

**Discussion by lucuo** [idaholab/moose#27211] [Q&A Modules: Solid mechanics] [answered]
I am now learning the XFEM program. I would like to know whether I can obtain the input file for the demonstration calculation case of XFEM（such as the case below） on MOOSE's official website, or where I can obtain it? Can anyone answer that for me? Thank you
![屏幕截图 2024-03-28 212456](https://github.com/idaholab/moose/assets/114867841/7eddb77a-fb93-4f9a-a0e4-66b11b5eb46c)

**Comment by GiudGiud** [comment]
@hugary1995

**Reply by jiangwen84** [reply]
@lucuo I made this simulation a couple of years ago. Unfortunately, I did not keep the input file. @lynnmunday has been recently working on the XFEM work, and he might be able to provide you an input file of similar or simpler problem.

**Reply by lucuo** [reply]
If you can provide a similar case, it will be of greater help to my study，your support is greatly appreciated.

**Reply by lynnmunday** [reply, marked as answer]
@lucuo the example you show is a complicated one to learn from.  A test like this one in the xfem module is simpler: 
 moose/modules/xfem/test/tests/solid_mechanics_basic/edge_crack_2d_propagation_mhs.i
Or the tests in this directory:
moose/modules/xfem/test/tests/nucleation_uo

The example you want is available in this git repository:  https://github.com/lynnmunday/xfem_pellet/tree/main
let me know if you can get it.

**Reply by lucuo** [reply]
Thank you very much for providing me with this case
